CRM/Core/BAO/CustomField.php

```diff
@@ -1492,15 +1495,21 @@ public static function formatCustomField(
       }
     }
 
-    $customFields = CRM_Core_BAO_CustomField::getFields($customFieldExtend,
-      FALSE,
-      $inline,
-      $customDataSubType,
-      NULL,
-      FALSE,
-      FALSE,
-      $checkPermission ? CRM_Core_Permission::EDIT : FALSE
-    );
+    $customFields = [];
+    $filters = [
+      'extends' => $customFieldExtend,
+      'is_active' => TRUE,
+    ];
+    if ($inline) {
+      $filters['style'] = 'Inline';
+    }
+    $customFieldGroups = CRM_Core_BAO_CustomGroup::getAll($filters, $checkPermission ? CRM_Core_Permission::EDIT : NULL);
+    foreach ($customFieldGroups as $customFieldGroup) {
+      foreach ($customFieldGroup['fields'] as $customFieldID => $customField) {
+        $customField['is_multiple'] = $customFieldGroup['is_multiple'];
+        $customFields[$customFieldID] = $customField;
+      }
+    }
```

Comment:
This part of the PR looks like general cleanup and I'd like to tackle this separately because this function is extremely messy & this cleanup could go a bit farther. Can you remove this bit of the PR for now?